File: Src/Visual Studio/SDK/C#/Eruru.ChatRobotRPC For Shared Project/Informations/ChatRobotGroupInformation.cs
namespace Eruru.ChatRobotRPC {

	/// <summary>
	/// 群信息
	/// </summary>
	public class ChatRobotGroupInformation {

		/// <summary>
		/// 群号
		/// </summary>
		public long Group { get; set; }
		/// <summary>
		/// 群名
		/// </summary>
		public string Name { get; set; }
		/// <summary>
		/// 群主QQ
		/// </summary>
		public long Master { get; set; }
		/// <summary>
		/// 是否是管理员（群主时也为真,可通过群主QQ区分）
		/// </summary>
		public bool IsAdministrator { get; set; }

	}

}

File: Src/Intelij IDEA/SDK/Java/Eruru.ChatRobotRPC/src/main/java/org/eruru/chatrobotrpc/informations/ChatRobotGroupInformation.java
package org.eruru.chatrobotrpc.informations;

/**
 * 群信息
 */
public class ChatRobotGroupInformation {

	private long group;
	private String name;
	private long master;
	private boolean isAdministrator;

	/**
	 * 群号
	 */
	public long getGroup () {
		return group;
	}

	/**
	 * 群号
	 */
	public void setGroup (long group) {
		this.group = group;
	}

	/**
	 * 群名
	 */
	public String getName () {
		return name;
	}

	/**
	 * 群名
	 */
	public void setName (String name) {
		this.name = name;
	}

	/**
	 * 群主QQ
	 */
	public long getMaster () {
		return master;
	}

	/**
	 * 群主QQ
	 */
	public void setMaster (long master) {
		this.master = master;
	}

	/**
	 * 是否是管理员（群主时也为真,可通过群主QQ区分）
	 */
	public boolean isAdministrator () {
		return isAdministrator;
	}

	/**
	 * 是否是管理员（群主时也为真,可通过群主QQ区分）
	 */
	public void setAdministrator (boolean administrator) {
		isAdministrator = administrator;
	}
}
